File: app/src/main/java/me/aungkooo/imageslider/HeaderItemView.java
package me.aungkooo.imageslider;

import android.content.Context;
import android.support.annotation.NonNull;
import android.support.v7.widget.CardView;
import android.view.LayoutInflater;
import android.widget.FrameLayout;
import android.widget.ImageView;
import android.widget.TextView;

import me.aungkooo.imageslider.databinding.ViewHeaderBinding;

public class HeaderItemView extends FrameLayout {

    @NonNull
    private final ViewHeaderBinding binding;

    public HeaderItemView(@NonNull Context context) {
        super(context);
        binding = ViewHeaderBinding.inflate(LayoutInflater.from(context), this, true);
    }

    @NonNull
    CardView getCardViewHeader() {
        return binding.cardViewHeader;
    }

    @NonNull
    ImageView getImageHeader() {
        return binding.imgHeader;
    }

    @NonNull
    TextView getTextHeader() {
        return binding.txtHeader;
    }
}

File: app/src/main/java/me/aungkooo/imageslider/HeaderItemView.kt
package me.aungkooo.imageslider

import android.content.Context
import android.view.LayoutInflater
import android.widget.FrameLayout
import android.widget.ImageView
import android.widget.TextView
import androidx.cardview.widget.CardView
import me.aungkooo.imageslider.databinding.ViewHeaderBinding

class HeaderItemView(context: Context) : FrameLayout(context) {

    private val binding: ViewHeaderBinding = ViewHeaderBinding.inflate(LayoutInflater.from(context), this, true)

    val cardViewHeader: CardView
        get() = binding.cardViewHeader
    val imageHeader: ImageView
        get() = binding.imgHeader
    val textHeader: TextView
        get() = binding.txtHeader

}